src/actix/api/update_api.rs

```diff
@@ -30,17 +30,19 @@ struct FieldPath {
 }
 
 #[derive(Deserialize, Serialize, JsonSchema, Validate)]
-pub struct UpdateParam {
-    pub wait: Option<bool>,
-    pub ordering: Option<WriteOrdering>,
+pub struct UpdateParams {
```

Comment:
This will cause changes in OpenAPI

Reply:
It doesn't because these params are explicitly defined. The consistency check CI job passes :heavy_check_mark:

Reply:
Name of the structure in Rust vs name of the structure if OpenAPI doesn't match?

Reply:
It's not even referenced by OpenAPI. It's not in the `schema_generator`.

https://github.com/qdrant/qdrant/blob/master/openapi/openapi-points.ytt.yaml#L88-L99.

Reply:
why `#[derive(..., JsonSchema, ..)]` then?

Reply:
🤷‍♀️

Reply:
I've removed `JsonSchema` derive from `UpdateParams`, CI is still green.